source/module.c

```diff
@@ -931,6 +939,48 @@ static void s_handle_failed_callback(napi_env env, napi_value function, napi_sta
     }
 }
 
+napi_status aws_napi_create_external_arraybuffer(
+    napi_env env,
+    void *external_data,
+    size_t byte_length,
+    napi_finalize finalize_cb,
+    void *finalize_hint,
+    napi_value *result) {
+
+    napi_status external_buffer_status =
+        napi_create_external_arraybuffer(env, external_data, byte_length, finalize_cb, finalize_hint, result);
+
+    if (external_buffer_status != napi_ok) {
+
+        /* TODO: The enum `napi_no_external_buffers_allowed` is introduced in node14.
+         * Use it to determine if the function failed because of the external buffer support after bump
+         * minimal support to node 14
+         */
+        AWS_NAPI_LOGF_ERROR(
+            "napi_create_external_arraybuffer (in aws_napi_create_external_arraybuffer) failed with : %s",
+            aws_napi_status_to_str(external_buffer_status));
```

Comment:
In the current code, this is going to log at the ERROR level, even though the next `napi_create_arraybuffer()` call is likely to succeed

Customers get **very scared** by ERROR logs

Don't log unless both attempts to make the buffer fail.